README.adoc

```diff
@@ -272,7 +272,7 @@ Writing a blog entry is very easy. All you have to do is fill in a brief prolog
 
 === Filename
 
-To write a blog entry, create a new textile document with the file extension .textile in the blog/ folder. The name of the file is important. It determines the date of the blog entry and the URL. The syntax is:
+To write a blog entry, creates a new textile document with the file extension .textile in the blog/ folder. The name of the file is important. It determines the date of the blog entry and the URL. The syntax is:
```

Comment:
I'm not a native English speaker, so I might be wrong. But I think this should be "create" as it was before. 

```suggestion
To write a blog entry, create a new textile document with the file extension .textile in the blog/ folder. The name of the file is important. It determines the date of the blog entry and the URL. The syntax is:
```